opensquirrel/squirrel_ir.py

```diff
@@ -24,9 +24,15 @@ def visit_float(self, f: "Float"):
     def visit_qubit(self, qubit: "Qubit"):
         pass
 
+    def visit_bit(self, bit: "Bit"):
+        pass
+
```

Comment:
This is only required if we don't deprecate the antlr parser, which we will, so it's not needed.